src/Jackett/Indexers/HDTorrents.cs

```diff
@@ -0,0 +1,233 @@
+        }
+
+        async Task<ReleaseInfo[]> PerformQuery(TorznabQuery query, string baseUrl)
+        {
+            List<ReleaseInfo> releases = new List<ReleaseInfo>();
+            List<string> searchurls = new List<string>();
+
+            foreach (var title in query.ShowTitles ?? new string[] { string.Empty })
+            {
+                var searchString = title + " " + query.GetEpisodeSearchString();
+                for (int page = 0; page < MAXPAGES; page++)
+                    searchurls.Add(string.Format(SearchUrl, HttpUtility.UrlEncode(searchString.Trim()), page));
+            }
+
+            foreach (string SearchUrl in searchurls)
+            {
+                var results = await client.GetStringAsync(SearchUrl);
+                try
+                {
+                    CQ dom = results;
+                    ReleaseInfo release;
+
+                    int rowCount = 0;
+                    var rows = dom[".mainblockcontenttt > tbody > tr"];
+                    foreach (var row in rows)
+                    {
+                        CQ qRow = row.Cq();
+                        if (rowCount < 2 || qRow.Children().Count() != 12) //skip 2 rows because there's an empty row & a title/sort row
+                        {
+                            rowCount++;
+                            continue;
+                        }
+
+                        release = new ReleaseInfo();
+                        long imdbid;
+                        long? size;
+                        int seeders, peers;
+
+                        release.Title = qRow.Find("td.mainblockcontent b a").Text();
+                        release.Description = release.Title;
+
+                        if (0 != qRow.Find("td.mainblockcontent u").Length && long.TryParse(qRow.Find("td.mainblockcontent u").Parent().First().Attr("href").Replace("http://www.imdb.com/title/tt", "").Replace("/", ""), out imdbid))
+                            release.Imdb = imdbid;
+
+                        release.MinimumRatio = 1;
+                        release.MinimumSeedTime = 172800;
+
+                        release.MagnetUri = new Uri(DefaultUrl + "/" + qRow.Find("td.mainblockcontent").Get(3).FirstChild.GetAttribute("href"));
+
+                        if (int.TryParse(qRow.Find("td").Get(9).FirstChild.FirstChild.InnerText, out seeders))
+                            release.Seeders = seeders;
+                        if (int.TryParse(qRow.Find("td").Get(10).FirstChild.FirstChild.InnerText, out peers))
+                            release.Peers = peers;
+
+                        string fullSize = qRow.Find("td.mainblockcontent").Get(6).InnerText;
+                        string[] sizeSplit = fullSize.Split(' ');
+                        switch (sizeSplit[1].ToLower())
+                        {
+                            case "kb":
+                                size = ReleaseInfo.BytesFromKB(float.Parse(sizeSplit[0], CultureInfo.InvariantCulture));
```

Comment:
Can you update all float.Parse, int.Parse, etc.. to use ParseUtil.Coerce..? I just implemented it yesterday to deal with all the number parsing bug reports we were getting. 
https://github.com/zone117x/Jackett/blob/master/src/Jackett/ParseUtil.cs